velox/docs/functions/spark/decimal.rst

```diff
@@ -43,6 +57,20 @@ precision and scale are adjusted.
     precision = 38
     scale = max(38 - (p - s), min(s, 6))
 
+Caps p at 38 and the scale should not be reduced when not allowing precision loss.
+For decimal addition, subtraction, multiplication, the adjustment logic is same.
+
+::
+    precision = 38
+    scale = min(38, s)
```

Comment:
This part does not render correctly. Perhaps one empty line after L63 is needed.